conf.py

```diff
@@ -150,7 +150,12 @@ def codif_correction_cmd_linux(x):
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "logo.png"
+project_path = pathlib.Path(__file__).parent.parent.resolve()
+project_path_string = project_path.name
+if project_path_string.__contains__("RTD-UM") or project_path_string.__contains__("RTD_UM"):
```

Comment:
```suggestion
if "RTD-UM" in project_path_string or "RTD_UM" in project_path_string:
```
Es preferible no utilizar métodos privados salvo que no quede otra opción.